packages/client/components/ActionMeetingSidebar.tsx

```diff
@@ -51,22 +63,21 @@ const ActionMeetingSidebar = (props: Props) => {
               gotoStageId(itemStageId).catch()
               handleMenuClick()
             }
+            const phaseCount = phaseType === NewMeetingPhaseTypeEnum.agendaitems && agendaItems
+              ? <StyledBadge>{agendaItems.length}</StyledBadge>
+              : undefined
             return (
               <NewMeetingSidebarPhaseListItem
                 key={phaseType}
                 phaseType={phaseType}
-                listPrefix={String(idx + 1)}
                 isActive={
                   phaseType === NewMeetingPhaseTypeEnum.agendaitems
                     ? blackList.includes(localPhaseType)
                     : localPhaseType === phaseType
                 }
-                isFacilitatorPhaseGroup={
-                  facilitatorPhaseType === phaseType ||
-                  (phaseType === NewMeetingPhaseTypeEnum.agendaitems &&
-                    blackList.includes(facilitatorPhaseType))
-                }
+                isUnsyncedFacilitatorPhase={isUnsyncedFacilitatorPhase && phaseType === facilitatorPhaseType}
                 handleClick={canNavigate ? handleClick : undefined}
+                meta={phaseCount}
```

Comment:
`meta` could be a little more expressive.
maybe call it `stageCount` have it take `undefined | null` to be hidden or a number to show?

Reply:
Sounds good.